code/modules/surgery/healing.dm

```diff
@@ -37,7 +37,8 @@
 	time = 25
 	var/brutehealing = 0
 	var/burnhealing = 0
-	var/missinghpbonus = 0 //heals an extra point of damager per X missing damage of type (burn damage for burn healing, brute for brute). Smaller Number = More Healing!
+	var/brutemultiplier = 0 //multiplies the damage that the patient has. if 0 the patient wont get any additional healing from the damage he has.
```

Comment:
use better variable names, for example missinh_hp_bonus instead of missinghpbonus, its much more readable that way